tests/v2/validation/provisioning/resources/cisbenchmark/cisbenchmark.go

```diff
@@ -96,5 +127,59 @@ func RunCISScan(client *rancher.Client, projectClusterID, scanProfileName string
 
 	logrus.Infof("CIS Benchmark scan passed!")
 
+	if printFailedChecks {
+		logrus.Infof("Checking cluster scan report for failed checks")
+		err = checkScanReport(steveclient, scan.Name)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func checkScanReport(steveClient *v1.Client, scanName string) error {
+	scanReportList, err := steveClient.SteveType(clusterScanReportType).List(nil)
+	if err != nil {
```

Comment:
is there no way to query for the scanName instead of looping through all reports?

Reply:
ClusterScanObject does not have report name in it. The cluster scan report name is randomly generated. It does not have field to identify with which cluster scan, it is associated. Hence, the fieldSelector cannot be used here and needs loop.

Reply:
ah makes sense. thanks for letting me know. I guess there isn't a way to pass in the query itself to check if the name contains the scanReportName right?